### DISCUSSION TITLE
Live collaboration in editor

### DISCUSSION THREAD

**Discussion by aqilc** [ppy/osu#13046] [Ideas]
**Describe the new feature:** 
I would love a feature that lets multiple people work on the map at the same time and seeing all the changes each other are making in real-time. This feature would greatly impact the mapping community as people don't have to take turns working on maps and then sending them on a chatting platform like Discord. Most of the mapping community would also greatly appreciate having this feature.

**Proposal designs of the feature:** 
All users can see what each of them have done real-time. Everyone can talk to each other using the chatting systems already in place. The person who originally made the map can add these users as Collaborators on their map and each Collaborator gets the map folder with every map imported to Lazer the second they log in.

**Comment by peppy** [comment]
I'd advise against making proposals for new features until we finish the game. Prerequisites involve:

- Finishing the game
- Finishing the editor
- #5379

**Comment by aqilc** [comment]
Sorry, but I have random ideas sometimes, and would like to express them before I forget everything about them.
